lib/Screens/AuthRegistration/auth_registartion_screen.dart

```diff
@@ -1,12 +1,18 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-import 'package:demo/Screens/AuthRegistration/body.dart';
+//import 'package:demo/Screens/AuthRegistration/components/body.dart';
+import 'package:demo/Screens/AuthRegistration/components/body_refactored.dart';
 
-class AuthRegistrationScreen extends StatelessWidget {
+class AuthRegistrationScreen extends StatefulWidget {
+  @override
+  _AuthRegistrationScreen createState() => _AuthRegistrationScreen();
+}
+
+class _AuthRegistrationScreen extends State<AuthRegistrationScreen> {
```

Comment:
Do you need a state here?

Reply:
No need! Corrected into `StatelessWidget!`